src/ripple/ledger/impl/View.cpp

```diff
@@ -44,6 +44,19 @@ bool amendmentRIPD1141 (NetClock::time_point const closeTime)
     return closeTime > amendmentRIPD1141SoTime();
 }
 
+NetClock::time_point const& amendmentRIPD1274SoTime ()
+{
+    using namespace std::chrono_literals;
+    // Thu Sept 8, 2016 10:00:00am PDT
+    static NetClock::time_point const soTime{526669200s};
```

Comment:
That is the point of a switch over. I'll talk to Warren to get an appropriate date.